packages/ui/src/views/w3m-desktop-connector-view/index.ts

```diff
@@ -48,7 +48,7 @@ export class W3mDesktopConnectorView extends LitElement {
     }
   }
 
-  private async onConnect() {
+  private async createConnectionAndWait(retry = 0) {
```

Comment:
Just trying to catch up: what is this solving?

Reply:
retries? There was a bug with sign v2 implementation that resulted in infinite fails / re-attempts. So I added this in as pre-caution for future (this bug was fixed, but kept this here just in case)